maven-plugin-plugin/src/it/skip/src/main/java/org/apache/maven/plugin/coreit/FirstMojo.java

```diff
@@ -18,47 +18,15 @@
  */
 package org.apache.maven.plugin.coreit;
 
-import java.io.File;
-
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
 
 /**
  * Touches a test file.
- *
- * @goal first
- * @requiresDependencyResolution test
- * @phase integration-test
- * @execute phase="generate-sources" lifecycle="cobertura"
- * @deprecated Don't use!
- * @since 1.2
  */
+@Mojo(name = "test")
 public class FirstMojo extends AbstractMojo {
```

Comment:
This is now completely different, how?

Reply:
Those mojos are just built and used to verify a few specific things depending on each test.  The annotations are usually not used or tested.